.github/workflows/sdk_android_build.yml

```diff
@@ -38,6 +38,10 @@ jobs:
       uses: actions/checkout@v4
       with:
         submodules: 'recursive'
+    - uses: actions/setup-java@v4
+      with:
+      distribution: 'oracle'
+      java-version: '17'
```

Comment:
Indentation here looks incorrect.  Are you mixing spaces and tabs?

Reply:
I think it's correct now.